src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic_Runtime.codegen.cs

```diff
@@ -33,21 +33,24 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_Basic : gl
             WriteLiteral(">\r\n    Hello world\r\n    ");
             EndContext();
             BeginContext(54, 29, false);
-#line (3,6)-(3,35) 6 "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic.cshtml"
-Write(string.Format("{0}", "Hello"));
+            Write(
+#line (3,6)-(3,35) "TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic.cshtml"
+string.Format("{0}", "Hello")
```

Comment:
Not in this case because we emitted the correct offset for the `Write(` call, but if you have multiple items on the same line then there was no way of distinguishing the user and generated parts of the code (the 'nested line directive issue' we've talked about in other threads).

I updated a test to show the effect here: https://github.com/dotnet/razor/pull/10006/commits/585d93ca8ad389566fc96daff978e66ce44a22ab#diff-0f7d2dbaf7d082ca0b57b2ee292ae388ee66a13caaf0e6360a367960dfc4afb5R50

The three emitted pieces are on multiple lines now in the output, but all map back to a single line in the original doc. Previously there was no way to tell if these were a single statement or separate (and if there is any generated code between them)

Reply:
I see, thanks.

Reply:
> the 'nested line directive issue' we've talked about in other threads

Do we want to close the issue by this PR?

Reply:
I checked and I don't think we have an issue, it was just a discussion in an email thread with an external customer.